lib/rubocop/cop/rspec/capybara/specific_matcher.rb

```diff
@@ -56,7 +79,31 @@ def specific_matcher(arg)
           end
 
           def acceptable_pattern?(arg)
-            arg.match?(/\[.+=\w+\]/) || arg.match?(/[ >,+]/)
+            arg.match?(/[ >,+]/)
+          end
+
+          def specific_matcher_option?(arg, matcher)
+            # If `button[foo-bar_baz=foo][bar][baz]`:
+            # extract "foo-bar_baz][bar][baz"
+            attributes, = arg.match(/\[([\w\W]*)\]/)&.captures
+            return true unless attributes
+
+            attributes.split('][').all? do |attr|
```

Comment:
How about e.g.
```suggestion
            # If `button[foo-bar_baz=foo][bar][baz]`:
            # extract ["foo-bar_baz", "bar", "baz"]
            attributes = arg.scan(/\[(.*?)\]/).flatten
            return true if attributes.empty?

            attributes.all? do |attr|
```

Reply:
I was having a little trouble there, but it looks to have improved  and better way to me. Thank you so much.